packages/app/styles/NavBar.css

```diff
@@ -0,0 +1,109 @@
+@import "~antd/dist/antd.css";
+
```

Comment:
![Screen Shot 2020-06-11 at 5 38 46 PM](https://user-images.githubusercontent.com/33739155/84452338-6f966880-ac0a-11ea-81fc-d8069da7e0b4.png)
When I moved the screen to a smaller size, I saw that the items were stacking before the drawer button was shown. The [Ant design menu](https://ant.design/components/menu/) might be able to help with that.

Reply:
Stanley and I agreed not to use the Ant design menu because it wasn't reactive. That's way there is a css file. I will try to fix the spacing when the screen is in that awkward size.

Reply:
Sounds good